clients/venice-thin-client/src/main/java/com/linkedin/venice/client/schema/RouterBasedStoreSchemaFetcher.java

```diff
@@ -81,8 +81,7 @@ public Schema getLatestValueSchema() {
   }
 
   private Schema getLatestValueSchemaFromAllValueSchemas() {
```

Comment:
StoreSchemaFetcher is mainly used by write compute, so based on this change, I assume you are planning to use `SchemaReader` for `online producer` instead of `StoreSchemaFetcher`, right?

Reply:
Yes. Online producer will not use `StoreSchemaFetcher`. This change only removes the need to pass the query path to the function as an argument. There are no functional changes in `RouterBasedStoreSchemaFetcher`